frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/install.tsx

```diff
@@ -99,11 +99,13 @@ export const CreateAttachedDevicesCluster: React.FC<CreateAttachedDevicesCluster
       match={match}
       lsoNs={lsoNs}
       mode={mode}
+      supportedFeatures={supportedFeatures}
     />
   );
 };
 
 type CreateAttachedDevicesClusterProps = {
   match: RouterMatch<{ appName: string; ns: string }>;
   mode: string;
+  supportedFeatures?: string[];
```

Comment:
Why is this optional ?

Reply:
This component will work without it as well. Not including it will not break it.

Reply:
This is used only in install.tsx file.
We dont need it anywhere else.

Reply:
resolved in the DM